### PR TITLE
ft_watcher: parse settleAuctionNoneLocal

### PR DESCRIPTION
**Pull request by bingyuyap**
This PR does a few things:
- parses `settle_auction_none_*` 
- add error handling for when ankr rpc nodes throw `unsafeRes` error on devnet
- tests for parsing. we cannot find any transaction with `settle_auction_none_cctp` so we shall not add the test case for that yet.

### CODE REVIEW

File: watcher/src/watchers/FTSolanaWatcher.ts [hunk 1]
```diff
@@ -671,17 +689,67 @@ export class FastTransferSolanaWatcher extends SolanaWatcher {
...
+    };
+
+    await this.saveFastTransferInfo('fast_transfer_settlements', info);
+    await this.updateMarketOrder({
+      fast_vaa_hash,
+      status: FastTransferStatus.SETTLED,
+    });
+
+    return {
+      id: {
```
Comment (panoel) [inline watcher/src/watchers/FTSolanaWatcher.ts:717]: Same comment as above.

File: watcher/src/watchers/FTSolanaWatcher.ts [hunk 2]
```diff
@@ -671,17 +689,67 @@ export class FastTransferSolanaWatcher extends SolanaWatcher {
...
     res: VersionedTransactionResponse,
-    instruction: MessageCompiledInstruction
+    ix: MessageCompiledInstruction
   ) {
-    // Slow relay is not done yet, will implement after
-    throw new Error('[parseSettleAuctionNoneCctp] not implemented');
+    const accountKeys = await this.getAccountsByParsedTransaction(res.transaction.signatures[0]);
+    const accountKeyIndexes = ix.accountKeyIndexes;
+    const payer = accountKeys[accountKeyIndexes[0]];
+    const auction = accountKeys[accountKeyIndexes[8]];
```
Comment (panoel) [inline watcher/src/watchers/FTSolanaWatcher.ts:731]: Is accountKeyIndexes guaranteed to have, at least, 9 elements?

File: watcher/src/watchers/FTSolanaWatcher.ts [hunk 3]
```diff
@@ -671,17 +689,67 @@ export class FastTransferSolanaWatcher extends SolanaWatcher {
...
     res: VersionedTransactionResponse,
     ix: MessageCompiledInstruction
-  ) {
-    // Slow relay is not done yet, will implement after
-    throw new Error('[parseSettleAuctionNoneLocal] not implemented');
+  ): Promise<{ id: FastTransferId; info: FastTransferSettledInfo }> {
+    const accountKeys = await this.getAccountsByParsedTransaction(res.transaction.signatures[0]);
+    const accountKeyIndexes = ix.accountKeyIndexes;
+    const payer = accountKeys[accountKeyIndexes[0]];
+    const auction = accountKeys[accountKeyIndexes[6]];
```
Comment (panoel) [inline watcher/src/watchers/FTSolanaWatcher.ts:696]: Is accountKeyIndexes guaranteed to have, at least, 7 elements?

File: watcher/src/watchers/FTSolanaWatcher.ts [hunk 4]
```diff
@@ -650,6 +664,10 @@ export class FastTransferSolanaWatcher extends SolanaWatcher {
     };
 
     await this.saveFastTransferInfo('fast_transfer_settlements', info);
+    await this.updateMarketOrder({
+      fast_vaa_hash,
+      status: FastTransferStatus.SETTLED,
+    });
 
     return {
       id: { fast_vaa_hash },
```
[the remark this answers is not in the document]
Comment (bingyuyap) [reply watcher/src/watchers/FTSolanaWatcher.ts:673]: you are right, this was left from previous PR. will update to return only info

File: watcher/src/watchers/FTSolanaWatcher.ts (hunk 2 above)
Comment (bingyuyap) [reply watcher/src/watchers/FTSolanaWatcher.ts:731]: according to idl, for this instructions yes. but i will add a check to make sure

**Review by panoel — APPROVED** [review]
LGTM
I like all the try catch blocks you added around the db

**Pull request merged**